alphastats/gui/AlphaPeptStats.py

```diff
@@ -17,7 +17,7 @@
 
 runtime = get_instance()
 session_id = get_script_run_ctx().session_id
-session_info = runtime._session_mgr.get_session_info(session_id)
+# session_info = runtime._session_mgr.get_session_info(session_id)
```

Comment:
Yes, this line causes the testing framework to fail and so far I haven't found the purpose otherwise. If we really don't need the line we can delete it at some point.

Reply:
if we don't use `session_info` anywhere, you can also just delete it ;-)